packages/strapi-plugin-users-permissions/config/schema.graphql.js

```diff
@@ -6,10 +6,10 @@ const _ = require('lodash');
  * @throws ApolloError if the body is a bad request
  */
 function checkBadRequest(contextBody) {
-  if (_.get(contextBody, 'output.payload.statusCode', 200) !== 200) {
-    const statusCode = _.get(contextBody, 'output.payload.statusCode', 400);
-    const message = _.get(contextBody, 'output.payload.message', 'Bad Request');
-    throw new Error(message, statusCode, _.omit(contextBody, ['output']));
+  if (_.get(contextBody, 'statusCode', 200) !== 200) {
+    const statusCode = _.get(contextBody, 'statusCode', 400);
+    const message = _.get(contextBody, 'message[0].messages[0].message', 'Bad Request');
+    throw new Error(message, statusCode, contextBody);
```

Comment:
Here the 2nd et 3rd params are not used. Can you create the Error object and set  a code property and a data property :)

Reply:
Done in f1710292431f69183bf44e7908e529814c9ea008